docs/modules/ROOT/partials/exoscale/environment-vars.adoc

```diff
@@ -1,9 +1,26 @@
 .Access to cloud API
 [source,bash]
 ----
-export EXOSCALE_ACCOUNT=<exoscale-account>
```

Comment:
I guess this was never used in the rest of the guide?

Reply:
This probably was a leftover which didn't get removed in 9f7a7a376953643152fe724554cbde7a8320cd0e when we removed the instructions to configure the Exoscale CLI via config file because we didn't think it was necessary (I probably only tested with `ch-dk-2` for `EXOSCALE_REGION`, so I didn't notice that that variable didn't have the desired effect).